src/components/29-timeline/timeline--right-dates.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="rvt-timeline rvt-timeline--right">
+  <div class="rvt-timeline__item">
+    <div class="rvt-timeline__marker" aria-hidden>
+    </div>
+    <div class="rvt-timeline__content">
+      <h2>Sample Item</h2>
+      <span class="rvt-timeline__date rvt-timeline__date--label">Spring 2019</span>
```

Comment:
On the second item in the timeline the date sets up against the item marker (blue dot). See attached screenshot.

![image](https://user-images.githubusercontent.com/5590034/61814368-f0dbe580-ae15-11e9-98ef-fd51c63d4506.png)